github/files/drupal7/scripts/devops/reinstall/vars/environments/bt-dev_env.yml

```diff
@@ -0,0 +1,13 @@
+---
+env:
+  pre_settings:
```

Comment:
We need to have more flexible config injector for been able to replace old configs, even if they were written in bad PHP style.
Also there is a need to remove closing php ```?>``` tag from settings file